adapter-http/src/main/resources/activities/documents-api/http-docsapi-crud-basic.yaml

```diff
@@ -1,6 +1,6 @@
 min_version: "4.17.15"
 
-# nb -v run driver=http yaml=http-docsapi-crud-basic tags=phase:schema docsapi_host=my_docsapi_host auth_token=$AUTH_TOKEN
+# nb -v run driver=http yaml=http-docsapi-crud-basic tags=block:"schema.*" docsapi_host=my_docsapi_host auth_token=$AUTH_TOKEN
```

Comment:
We should remove all such "examples" since the named scenarios are a better place for this.

Reply:
We did discuss that, good catch.